### PR TITLE
Add docs about using AWS CLI version 2 Docker image

### PR DESCRIPTION
**Pull request by lucasepe**
Add documentation about how to setup Localstack `docker-compose.yml` file to use  the official AWS CLI v2 docker container. (see #3088)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -319,6 +319,77 @@ awslocal kinesis list-streams
...
+
+```yml
+version: "3.8"
+
+networks:
+  default:
+    external:
+      name: "localstack"
+
+services:
```
Comment (whummer) [inline README.md:342]: Thanks for extending the docs @lucasepe . I think a few sections are a bit redundant - can we try to trim it down to the bare minimum (and avoid duplication with other parts of the README)? I think (at least) lines 335-336, 341-360, 363-368 are redundant and can probably be removed. Thanks!

File: README.md [hunk 2]
```diff
@@ -319,6 +319,77 @@ awslocal kinesis list-streams
 **UPDATE**: Use the environment variable `$LOCALSTACK_HOSTNAME` to determine the target host
 inside your Lambda function. See [Configurations](#Configurations) section for more details.
 
+## Using the official AWS CLI version 2 Docker image with Localstack Docker container
+
+By default the the container running [amazon/aws-cli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-docker.html) is isolated from `0.0.0.0:4566` on the host machine, that means that aws-cli cannot reach localstack through your shell.
```
Comment (whummer) [inline README.md:324]: nitpick: `the the`

Comment (lucasepe) [reply README.md]: thanks!

**Comment by whummer** [comment]
Thanks for the quick turnaround. 👍

**Pull request merged**